src/components/StoreInfo/StoreLocation.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react'
+import styles from './styles.module.scss'
+import { Location } from '../../utilities';
+
+
+interface Props {
+  locations : Location[] | undefined
+}
+
+const StoreLocation: FC<Props> = ({ locations }) => {
```

Comment:
Did you run `yarn format`? The indenting here looks to be too much

Reply:
Yeah I missed this, think I mostly just hesitate to run it since it looks like it runs on every typescript file so I just run it on individual files. Do we have a pre-commit hook for running Prettier btw?

Reply:
we don't have a pre-commit hook right now, but that's probably a good idea to implement!